backend/Api/Common/StorageService.cs

```diff
@@ -75,102 +92,95 @@ private List<Consultant> LoadConsultantsFromDb(string orgUrlKey)
         return hydratedConsultants;
     }
 
-    public void UpdateStaffing(int id, double hours)
+    private Core.DomainModels.Staffing CreateStaffing(StaffingKey staffingKey, double hours )
```

Comment:
Vil se om jeg klarer å fikse disse namespacene

Reply:
Ta det i en senere PR? 😄

Reply:
GPT Svar: 

It seems like you are experiencing an issue with how classes are referenced in your .NET project. The issue might be related to the namespace in which your classes are defined.

When you declare a class in C#, it can be part of a namespace. For example:

```csharp
namespace Core.DomainModels
{
    public class Staffing
    {
        // Class implementation
    }
}
```

If your class is defined within a namespace, you need to reference it using the full namespace path unless you have specified a using directive at the top of your file.

For example, if you have the following using directive:

```csharp
using Core.DomainModels;
```

You can then reference the `Staffing` class directly without using the full namespace:

```csharp
Staffing staffingInstance = new Staffing();
```

If you are still seeing the full namespace in your code, there are a few things you can check:

1. **Check your using directives:** Make sure that you have the appropriate using directives at the top of your file. For the example above, you should have `using Core.DomainModels;`.

2. **Verify the namespace in the class file:** Open the file where the `Staffing` class is defined and ensure that it is indeed in the `Core.DomainModels` namespace.

3. **Check for naming conflicts:** Make sure there are no naming conflicts in your project. If there is another class with the same name in a different namespace, you might get unexpected behavior.

If you've checked these things and are still experiencing issues, please provide more details, and I'll do my best to assist you further.

Prøver i 2 min å fikse det :)